moto/sagemaker/models.py

```diff
@@ -1105,6 +1105,8 @@ def gen_response_object(self) -> Dict[str, Any]:
             "AdditionalInferenceSpecifications",
             "SkipModelValidation",
         ]
+        if "ModelPackageGroupName" in response_object and response_object["ModelPackageGroupName"] == response_object["ModelPackageName"]:
+            del response_object["ModelPackageName"]
```

Comment:
This is what I've referred too as being too unconventional. For versioned models - i.e. Model Package Groups - the ModelPackageName is omitted from the output.